src/constants/index.ts

```diff
@@ -0,0 +1,2 @@
+export const MODE_ABSOLUTE = 'absolute';
+export const MODE_RELATIVE = 'relative';
```

Comment:
można by to przenieść do enum'a, nie musiałbyś wtedy robić tych typów, chyba, że chccesz zachować jakieś backward compatibility

Reply:
Chciałbym aby te constanty były importowalne w zwykłym JS, więc enum mnie średnio urządza tutaj.